src/Mvc/test/WebSites/SimpleWebSiteWithWebApplicationBuilder/FakeEntryPoint.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+/// <summary>
+/// This is a class we use to reference this assembly statically from tests
+/// </summary>
+namespace SimpleWebSiteWithWebApplicationBuilder
+{
+    public class FakeStartup
```

Comment:
This feels like a bummer. Also I imagine it's going to throw people off since `MvcTestFixture` expects a `TStartup`? Would users think they are unable to use top-level statements because of it's signature?

Reply:
> This feels like a bummer.

I'm in a discussion with the C# team to enable generating a public type from top level programs.

> Also I imagine it's going to throw people off since MvcTestFixture expects a TStartup

We don't ship MvcFixture right?

> Would users think they are unable to use top-level statements because of it's signature?

Yes. See the first comment.

Reply:
> We don't ship MvcFixture right?

Ahh right. `WebApplicationFactory` expects a `TEntryPoint` so that's definitely less problematic if the top level type can be referenced in code.